Formula/i/ioctl.rb

```diff
@@ -1,8 +1,9 @@
 class Ioctl < Formula
   desc "Command-line interface for interacting with the IoTeX blockchain"
   homepage "https://docs.iotex.io/"
-  url "https://github.com/iotexproject/iotex-core/archive/refs/tags/v1.14.0.tar.gz"
-  sha256 "7ba82d03f51f46355230c14f9d68e523130dd6706d255cda6aec81cbd73ff59e"
+  url "https://github.com/iotexproject/iotex-core/archive/refs/tags/v1.14.1-testing.tar.gz"
+  version "1.14.1-testing"
```

Comment:
Seems suspicious

Reply:
Yeah definitely. The actual "latest" is tagged as such, so we can use `github_latest` for livecheck.